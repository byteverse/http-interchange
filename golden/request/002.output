post /foo/bar?baz=true HTTP/1.1
Content-Type: application/json
Host: example.com

