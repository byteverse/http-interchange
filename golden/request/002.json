{
    "requestLine": {
        "method": "post",
        "path": "/foo/bar?baz=true",
        "versionMajor": 1,
        "versionMinor": 1
    },
    "headers": [
        {
            "name": "Content-Type",
            "value": "application/json"
        },
        {
            "name": "Host",
            "value": "example.com"
        }
    ]
}
